### PR TITLE
Removing private SSH key

### PR DESCRIPTION
**Pull request by oudeismetis**
I noticed you had commited a private ssh key to your repo. 

**"I know. It was intentional."** 
Sorry to bother you then. Feel free to ignore this PR. 
You know you can also [secure private data while keeping it under source control](https://keybase.io/blog/encrypted-git-for-everyone).

**"I didn't notice. What is this file? Is it a problem?"**
[This can be a very bad thing](http://www.securityweek.com/github-search-makes-easy-discovery-encryption-keys-passwords-source-code)

**"I keep accidentally typing `git add .`, any way I can prevent that from being an issue?"**
There are a couple of steps you should take immediately:
1. Avoid commands like `git add .` and `git add *`. It's better to be more explicit when you can.
1. Assume your key is compromised. Unassociate it from any accounts you are using it for and [generate a new key](https://help.github.com/articles/reviewing-your-ssh-keys/).
1. Store your keys in a location outside of your project.
1. Use a [.gitignore file](https://git-scm.com/docs/gitignore) to prevent accidental commits of certain files.

Github also has a page about how to [remove sensitive data from a repo](https://help.github.com/articles/removing-sensitive-data-from-a-repository/).

Hope this PR finds you well

### CODE REVIEW

**Comment by CAD97** [comment]
...... I thought I went through the [travis encrypt file](https://docs.travis-ci.com/user/encrypting-files/) process.

Luckily, following decent practice, this key is only keyed as a deploy key for this repository. I've removed it, so (RIP Travis builds, all going to fail now) there is no further risk; the key has been neutralized.

For the life of me, I was unable to get the Travis encryption working. Apparently the reason it started working was that I disabled it? Somehow? Even though Travis was decrypting it?

https://github.com/nafi-lang/rust-nafi/blob/e4e7ec044a49996b02ef0555590ce1210b2cd6de/.travis.yml#L4

Basically, Travis needs a way to write to the repository such that it can update the gh-pages branch with up-to-date generated documentation. Hopefully I can go through the process of encrypting the private key properly this time. It's unfortunate that the automatic way didn't work for me on Windows even though I tried multiple times.

**Comment by CAD97** [comment]
- CLA not required; removal only
- Travis is going to fail because it's removing a required key
- I'll get to fixing it this weekend hopefully

**Comment by CAD97** [comment]
(In hindsight, it may have been better to commit 30e7e0430ffc1eafbe10a1324b884c03a63a37b4 and then merge via bors... but oh well.)

**Comment by oudeismetis** [comment]
glad I could help!

**Comment by CAD97** [comment]
Noting this here so when I have this same problem again I might solve it faster. It took me a while longer than expected (life got in the way) but...

I got motivated to work on this again, and got the encryption right this time. https://github.com/nafi-lang/rust-nafi/blob/master/.travis/id_rsa.enc is actually an encrypted binary blob, and docs are auto-building to https://nafi-lang.github.io/rust-nafi/ again.

https://github.com/travis-ci/travis-ci/issues/4746 describes the issue on Windows. Unfortunately, since it's been around for two years, I don't think it's getting fixed anytime soon. Luckily, I stumbled across https://github.com/travis-ci/travis-ci/issues/8777 which allowed me to manually encrypt the file correctly this time.
